<ui version="4.0" stdsetdef="1" >
  <author/>
  <comment/>
  <exportmacro/>
  <class>buildNetDialog</class>
  <widget class="QDialog" name="buildNetDialog" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>299</width>
        <height>222</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Build a neural network</string>
    </property>
    <property name="sizeGripEnabled" >
      <bool>true</bool>
    </property>
    <layout class="QVBoxLayout" >
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <widget class="QLabel" name="NumInput" >
              <property name="text" >
                <string>Number of input neurons</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer4" >
              <property name="sizeHint" >
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QSpinBox" name="SpinBoxInput" >
              <property name="maximum" >
                <number>999</number>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <widget class="QLabel" name="NumHidden" >
              <property name="text" >
                <string>Number of hidden neurons</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer5" >
              <property name="sizeHint" >
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QSpinBox" name="SpinBoxHidden" >
              <property name="maximum" >
                <number>999</number>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <widget class="QLabel" name="NumOutput" >
              <property name="text" >
                <string>Number of output neurons</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer6" >
              <property name="sizeHint" >
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QSpinBox" name="SpinBoxOutput" >
              <property name="maximum" >
                <number>999</number>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <spacer name="spacer1" >
              <property name="sizeHint" >
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="buttonBuildOk" >
              <property name="text" >
                <string>Ok</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="buttonBuildCancel" >
              <property name="text" >
                <string>Cancel</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer1_2" >
              <property name="sizeHint" >
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  
  <tabstops>
    <tabstop>SpinBoxInput</tabstop>
    <tabstop>SpinBoxHidden</tabstop>
    <tabstop>SpinBoxOutput</tabstop>
    <tabstop>buttonBuildOk</tabstop>
    <tabstop>buttonBuildCancel</tabstop>
  </tabstops>
  <connections/>
</ui>
